core/setup.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+PACKAGE_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+with open(os.path.join(PACKAGE_ROOT, 'README.rst')) as file_obj:
```

Comment:
I disagree. It's YAGNI until we have a reason to put non-ASCII characters in an english language document. If it did end up erroring, our tag builds would fail to deploy and we'd notice.